  <launch>
  <!-- DVS simulator using a sequence of images -->
  <node name="" pkg="dvs_simulator_py" type="dvs_simulator.py" output="screen">
  
    <!-- Dataset parameters -->
    <param name="dataset_name" value="3planes" />
    
    <!-- DAVIS parameters -->
    <param name="contrast_threshold" value="0.15" />
    
    <param name="blur_size" value="3" />
    
    <param name="event_streaming_rate" value="60.0" />
    <param name="image_streaming_rate" value="40.0" />
    
    <param name="write_to_bag" value="True" />
    
  </node>
  
  <!-- visualization -->
  <node name="dvs_renderer" pkg="dvs_renderer" type="dvs_renderer" output="screen">
    <param name="display_method" value="red-blue"/>
    <remap from="events" to="/dvs/events" />
    <remap from="image" to="/dvs/image_raw" />
    <remap from="dvs_rendering" to="dvs_rendering" />
  </node>


  <!-- display -->
  <node name="image_view" pkg="image_view" type="image_view">
  	<remap from="image" to="dvs_rendering"/>
  </node>

</launch>
